runbotics/runbotics-desktop/src/action/variable/variable.action-handler.ts

```diff
@@ -38,11 +38,7 @@ export default class VariableActionHandler extends StatelessActionHandler {
     ): Promise<AssignVariableActionOutput> {
         const variables = {};
         variables[input.variable] = input.value;
-        // assign to local scope
-        const setVariables = { ...request.executionContext.environment.variables, ...variables };
-        request.executionContext.environment.assignVariables(setVariables);
 
-        // assign to global scope
```

Comment:
im not sure about it. debug what the difference between so called local and global scope